packages/rancher-alerting/rancher-sachet/charts/values.yaml

```diff
@@ -56,13 +56,12 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
-nodeSelector:
-  kubernetes.io/os: linux
-
-tolerations:
-  - key: cattle.io/os
-    operator: "Equal"
-    value: "linux"
-    effect: NoSchedule
+## Node labels for pod assignment
+## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+##
+nodeSelector: {}
+
+## List of node taints to tolerate (requires Kubernetes >= 1.6)
+tolerations: []
```

Comment:
already has support for using the helper template to auto-insert these tolerations and nodeSelectors, not sure why it was added twice